build-system/common/esbuild-babel.js

```diff
@@ -62,9 +62,9 @@ function getEsbuildBabelPlugin(
     const promise = babel
       .transformAsync(contents, babelOptions)
       .then((result) => {
-        const {code, map} = result || {};
-        debug('post-babel', filename, code, map);
-        return code + `\n// ${filename}`;
+        const {code} = result || {};
+        debug('post-babel', filename, code);
```

Comment:
Why'd you remove the map? I check this when doing babel changes to verify the full system.

Reply:
Is it still helpful with an inline sourcemap? I don't mind putting it back into the debug statement!

Reply:
I put the map back